catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -495,13 +495,21 @@ public Table createTable(
       IcebergColumn[] icebergColumns =
           Arrays.stream(columns)
               .map(
-                  column ->
-                      new IcebergColumn.Builder()
-                          .withName(column.name())
-                          .withType(column.dataType())
-                          .withComment(column.comment())
-                          .withNullable(column.nullable())
-                          .build())
+                  column -> {
+                    if (!column.defaultValue().equals(Column.DEFAULT_VALUE_NOT_SET)) {
```

Comment:
is it possible `column.defaultValue()` is null?

Reply:
using Preconditions seems more clear

Reply:
> is it possible `column.defaultValue()` is null?

no. The higher level entrances have handled this situation